### PR TITLE
feat: add custom operation in swagger fix #614

### LINKED ISSUE #614: Swagger 2.0 api documentation doesn't support custom operations.
How to reproduce:

```php
/**
 * @ApiResource(
 *     iri="https://schema.org/Foo",
 *     itemOperations={
 *          "bar": {"route_name": "api_foo_bar", "method": "GET"},
 *          "get": {"method": "GET"}
 *     }
 *     )
 */
```

Expected result:

2 operations in the swagger output, currently only the GET route is outputted.

Problem:
\ApiPlatform\Core\Swagger\ApiDocumentationBuilder::getSwaggerOperation

IMO the code before the switch statement is wrong.

### PR DESCRIPTION
**Pull request by Simperfit**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #614 
| License       | MIT
| Doc PR        |

### CODE REVIEW

File: src/Swagger/ApiDocumentationBuilder.php [hunk 1]
```diff
@@ -76,13 +76,17 @@ public function getApiDocumentation() : array
     {
         $classes = [];
         $operation = [];
+        $customOperation = [];
+
```
Comment (dunglas) [inline src/Swagger/ApiDocumentationBuilder.php]: Those 2 blank lines must be removed.

**Comment by dunglas** [comment]
Can you add a non-regression test please?

**Comment by Simperfit** [comment]
added a test

**Comment by dunglas** [comment]
Thank you @Simperfit

**Comment by samvdb** [comment]
Thank you very much @Simperfit ! Awesome work

**Comment by Simperfit** [comment]
Thanks both of you @dunglas @samvdb, that's nice ! ;)

**Pull request merged**